plugin/kubernetes/xfr.go

```diff
@@ -71,6 +75,25 @@ func (k *Kubernetes) Transfer(ctx context.Context, state request.Request) (int,
 	return dns.RcodeSuccess, nil
 }
 
+// transferAllowed checks if incoming request for transferring the zone is allowed according to the ACLs.
+func (k *Kubernetes) transferAllowed(state request.Request) bool {
```

Comment:
Please add a comment that this is copy from zone.transferAllowed and should be (at somepoint) factored into a transfer pkg (or where ever we put it)